### ISSUE TITLE
[1.4.1] Lag on getting/refreshing content

### ISSUE THREAD

**Issue by rancidfrog** [gitpoint/git-point#670]
App lags when refreshing content, displaying issues, ...
It is quick when displaying already cached items, but as soon as it tries to get new content there is horrible lag. 
Also, instagram seems to work fine and is much more of a resource hog than git-point, I noticed as I though it might be due to bad/slow connection.
For example, clicked on issue and app lags on ~
![](https://s31.postimg.org/x0h4i2xiz/Screenshot_20171219-122719_01.png)

**Comment by machour** [comment]
@rancidfrog could you give us some metrics here? How much seconds does it take to display this one for example: #432.

On my end, on a LTE connection, it takes 1.5 second (approx) which is quiet decent knowing that the issue screen is one of the heaviest in the app (up to 7 REST calls if I remember). This will be fixed by using GraphQL for this page.

**Comment by rancidfrog** [comment]
Opening this issue took ~9 seconds before it got the content.
Now reopening takes around 2~3seconds
432, did not time it but again now with content available it takes around 2~3seconds

**Comment by chinesedfan** [comment]
@rancidfrog Yes, our issue screen sends too many requests, unless GraphQL is introduced.

And comparing with instagram can't eliminate networks reasons. Maybe Github APIs perform slower than instagram. :)
